synapse_data_warehouse/synapse_raw/tasks/V2.25.1__update_node_tasks.sql

```diff
@@ -1,6 +1,6 @@
 -- Configure environment
 USE SCHEMA {{database_name}}.synapse_raw; --noqa: JJ01,PRS,TMP,CP02
-USE WAREHOUSE compute_xsmall;
```

Comment:
This isn't the script that requires compute medium. Let's chat more on monday

Reply:
Ah, okay. I thought maybe it needed it since we are probably loading over 100 million records.

Reply:
You have that here: https://github.com/Sage-Bionetworks/snowflake/blob/dev/synapse_data_warehouse%2Fsynapse_raw%2Ftables%2FV2.25.2__refresh_node_snapshot.sql#L2-L4

Reply:
I need to stop working 😓

Reply:
😅 you and me both. Almost the holidays.